package-res/resources/web/pentaho/data/CellTuple.js

```diff
@@ -26,25 +26,29 @@ define([
       this.base();
 
       // Fill with null valued cells.
-      var i = this.length, SL = this.structure.length;
+      var i = this.length;
+      var SL = this.structure.length;
       while(i++ < SL) this.add(null);
     },
 
-    //region IListElement
+    // region IListElement
     elemName: "CellTuple",
-    //endregion
+    // endregion
 
-    //region IWithKey implementation
+    // region IWithKey implementation
     keyName:  "key",
 
     _key: null,
 
     get key() {
       var key = this._key;
-      if(key == null) key = this._key = this.map(function(cell) { return cell.key; }).join("~");
+      if(key == null)
```

Comment:
I'm with Nelson on this one. As a single-line statement, it's clear that you can't add another statement inside the if bit without adding the bracket.